diff -up Clp-1.15.7/Makefile.in.orig Clp-1.15.7/Makefile.in
--- Clp-1.15.7/Makefile.in.orig	2014-07-16 12:35:34.683193514 -0300
+++ Clp-1.15.7/Makefile.in	2014-07-16 12:35:58.494194426 -0300
@@ -406,7 +406,7 @@ EXTRA_DIST = doc/authors.xml doc/basicmo
 ########################################################################
 pkgconfiglibdir = $(libdir)/pkgconfig
 pkgconfiglib_DATA = clp.pc $(am__append_3)
-addlibsdir = $(DESTDIR)$(datadir)/coin/doc/Clp
+addlibsdir = $(DESTDIR)$(datadir)/doc/coin-or-$(PACKAGE_NAME)
 
 ########################################################################
 #                         Maintainer Stuff                             #
@@ -416,7 +416,7 @@ CLEANFILES =
 # Files that are generated and should be cleaned with make distclean
 DISTCLEANFILES = $(am__append_5) $(VPATH_DISTCLEANFILES)
 DocFiles = README AUTHORS LICENSE
-DocInstallDir = $(datadir)/coin/doc/$(PACKAGE_NAME)
+DocInstallDir = $(datadir)/doc/coin-or-$(PACKAGE_NAME)
 all: all-recursive
 
 .SUFFIXES:
