--- Clp-1.17.3/src/ClpSimplexOther.cpp.orig	2019-01-07 12:04:36.000000000 -0700
+++ Clp-1.17.3/src/ClpSimplexOther.cpp	2019-06-19 09:35:35.100664914 -0600
@@ -2360,9 +2360,9 @@ int ClpSimplexOther::parametrics(const c
   if (intervalTheta >= endTheta)
     intervalTheta = 0.0;
   if (!good) {
-    sprintf(line, "Odd first line %s on file %s?", line, dataFile);
     handler_->message(CLP_GENERAL, messages_)
-      << line << CoinMessageEol;
+      << "Odd first line " << line << " on file " << dataFile << "?"
+      << CoinMessageEol;
     fclose(fp);
     return -2;
   }
