diff -up Clp-1.16.6/Makefile.in.orig Clp-1.16.6/Makefile.in
--- Clp-1.16.6/Makefile.in.orig	2015-04-12 12:58:41.513642557 -0300
+++ Clp-1.16.6/Makefile.in	2015-04-12 12:59:06.943643531 -0300
@@ -420,7 +420,7 @@ EXTRA_DIST = doc/authors.xml doc/basicmo
 ########################################################################
 pkgconfiglibdir = $(libdir)/pkgconfig
 pkgconfiglib_DATA = clp.pc $(am__append_3)
-addlibsdir = $(DESTDIR)$(datadir)/coin/doc/Clp
+addlibsdir = $(DESTDIR)$(datadir)/doc/coin-or-$(PACKAGE_NAME)
 
 ########################################################################
 #                         Maintainer Stuff                             #
@@ -430,7 +430,7 @@ CLEANFILES =
 # Files that are generated and should be cleaned with make distclean
 DISTCLEANFILES = $(am__append_5) $(VPATH_DISTCLEANFILES)
 DocFiles = README AUTHORS LICENSE 
-DocInstallDir = $(datadir)/coin/doc/$(PACKAGE_NAME)
+DocInstallDir = $(datadir)/doc/coin-or-$(PACKAGE_NAME)
 COIN_HAS_DOXYGEN = @COIN_HAS_DOXYGEN_TRUE@TRUE
 COIN_HAS_LATEX = @COIN_HAS_LATEX_TRUE@TRUE
 all: all-recursive
