diff -up Clp-releases-1.17.5/Clp/configure.orig Clp-releases-1.17.5/Clp/configure
--- Clp-releases-1.17.5/Clp/configure.orig	2020-02-03 06:32:17.000000000 -0700
+++ Clp-releases-1.17.5/Clp/configure	2020-02-20 09:46:50.382716225 -0700
@@ -32230,7 +32230,7 @@ echo "${ECHO_T}$doxysrcdir (src)" >&6
   fi
   # Not built, fall back to installed tag file
   if test $doxyfound = no ; then
-    eval doxydir="${datadir}/coin/doc/${proj}/doxydoc"
+    eval doxydir="${datadir}/doc/coin-or-${proj}"
     # AC_MSG_NOTICE([Considering $doxydir (install)])
     # AC_MSG_NOTICE([Subdirs: $coin_subdirs)])
     coin_doxy_tagfiles="$coin_doxy_tagfiles $doxydir/$doxytag=$doxydir/html"
diff -up Clp-releases-1.17.5/Clp/Makefile.in.orig Clp-releases-1.17.5/Clp/Makefile.in
--- Clp-releases-1.17.5/Clp/Makefile.in.orig	2020-02-03 06:32:17.000000000 -0700
+++ Clp-releases-1.17.5/Clp/Makefile.in	2020-02-20 09:46:50.398715939 -0700
@@ -421,7 +421,7 @@ EXTRA_DIST = doc/authors.xml doc/basicmo
 ########################################################################
 pkgconfiglibdir = $(libdir)/pkgconfig
 pkgconfiglib_DATA = clp.pc $(am__append_3)
-addlibsdir = $(DESTDIR)$(datadir)/coin/doc/Clp
+addlibsdir = $(DESTDIR)$(datadir)/doc/coin-or-$(PACKAGE_NAME)
 
 ########################################################################
 #                         Maintainer Stuff                             #
@@ -431,7 +431,7 @@ CLEANFILES =
 # Files that are generated and should be cleaned with make distclean
 DISTCLEANFILES = $(am__append_5) $(VPATH_DISTCLEANFILES)
 DocFiles = README AUTHORS LICENSE 
-DocInstallDir = $(datadir)/coin/doc/$(PACKAGE_NAME)
+DocInstallDir = $(datadir)/doc/coin-or-$(PACKAGE_NAME)
 COIN_HAS_DOXYGEN = @COIN_HAS_DOXYGEN_TRUE@TRUE
 COIN_HAS_LATEX = @COIN_HAS_LATEX_TRUE@TRUE
 all: all-recursive
