diff -up Clp-releases-1.17.5/Clp/src/ClpModel.cpp.orig Clp-releases-1.17.5/Clp/src/ClpModel.cpp
--- Clp-releases-1.17.5/Clp/src/ClpModel.cpp.orig	2020-02-03 06:32:17.000000000 -0700
+++ Clp-releases-1.17.5/Clp/src/ClpModel.cpp	2020-02-20 09:50:46.973499354 -0700
@@ -3720,7 +3720,7 @@ int ClpModel::emptyProblem(int *infeasNu
             numberDualInfeasibilities++;
             ;
             sumDualInfeasibilities += fabs(objValue);
-            returnCode |= 2;
+            returnCode |= 1;
           }
           status_[i] = 0;
         }
