diff -up Clp-releases-1.17.5/Clp/src/ClpSimplexOther.cpp.orig Clp-releases-1.17.5/Clp/src/ClpSimplexOther.cpp
--- Clp-releases-1.17.5/Clp/src/ClpSimplexOther.cpp.orig	2020-02-03 06:32:17.000000000 -0700
+++ Clp-releases-1.17.5/Clp/src/ClpSimplexOther.cpp	2020-02-20 09:55:43.777239225 -0700
@@ -2360,9 +2360,9 @@ int ClpSimplexOther::parametrics(const c
   if (intervalTheta >= endTheta)
     intervalTheta = 0.0;
   if (!good) {
-    sprintf(line, "Odd first line %s on file %s?", line, dataFile);
     handler_->message(CLP_GENERAL, messages_)
-      << line << CoinMessageEol;
+      << "Odd first line " << line << " on file " << dataFile << "?"
+      << CoinMessageEol;
     fclose(fp);
     return -2;
   }
