--- Clp-1.17.1/src/ClpSolver.cpp.orig	2019-03-27 11:53:43.018224450 -0600
+++ Clp-1.17.1/src/ClpSolver.cpp	2019-04-02 20:37:12.984912142 -0600
@@ -293,7 +293,11 @@ int ClpMain1(int argc, const char *argv[
   parameters[whichParam(CLP_PARAM_DBL_PRIMALWEIGHT, parameters)].setDoubleValue(models->infeasibilityCost());
   parameters[whichParam(CLP_PARAM_ACTION_RESTORE, parameters)].setStringValue(restoreFile);
   parameters[whichParam(CLP_PARAM_ACTION_SAVE, parameters)].setStringValue(saveFile);
+#ifdef COIN_HAS_CBC
+  parameters[whichParam(CBC_PARAM_DBL_TIMELIMIT_BAB, parameters)].setDoubleValue(models->maximumSeconds());
+#else
   parameters[whichParam(CLP_PARAM_DBL_TIMELIMIT, parameters)].setDoubleValue(models->maximumSeconds());
+#endif
   parameters[whichParam(CLP_PARAM_ACTION_SOLUTION, parameters)].setStringValue(solutionFile);
   parameters[whichParam(CLP_PARAM_ACTION_SAVESOL, parameters)].setStringValue(solutionSaveFile);
   parameters[whichParam(CLP_PARAM_INT_SPRINT, parameters)].setIntValue(doSprint);
